resources/views/livewire/domestic-address-search-bar.blade.php

```diff
@@ -1,20 +1,112 @@
-        <li
-            wire:click="setQuery({{ $contact }})"
+        @foreach ($results as $result)
+        <div
+            wire:click="selectResult({{ $result }})"
         >
-            {{ $contact }}
-        </li>
+            {{ $result->get('description') }}
+        </div>
         @endforeach
-    </ul>
-</div>
\ No newline at end of file
+    </div>
+    <form 
+        method="POST"
+        action=""
+        class="w-full"
+    >
+        @csrf
+        <div class="flex justify-between w-full mt-4 text-gray-700">
+            <div class="w-4/12">
+                <label class="ml-2 text-xs text-gray-500 font-semibold tracking-widest">
+                    Address 1
+                </label>
+                <input
+                    id="address-line-1"
+                    name="address-line-1"
+                    type="text"
+                    readonly="readonly"
+                    class="w-full px-2 py-1 bg-yellow-50"
+                >
+            </div>
+            <div class="w-4/12 ml-2">
+                <label class="ml-2 text-xs text-gray-500 font-semibold tracking-widest">
+                    Address 2
+                </label>
+                <input
+                    id="address-line-2"
+                    name="address-line-2"
+                    type="text"
+                    class="w-full px-2 py-1"
+                >
+            </div>
+            <div class="w-2/12 ml-2">
+                <label class="ml-2 text-xs text-gray-500 font-semibold tracking-widest">
+                    City
+                </label>
+                <input
+                    id="city"
+                    name="city"
+                    type="text"
+                    readonly="readonly"
+                    class="w-full px-2 py-1 bg-yellow-50"
+                >
+            </div>
+            <div class="w-1/12 ml-2">
+                <label class="ml-2 text-xs text-gray-500 font-semibold tracking-widest">
+                    State
+                </label>
+                <input
+                    id="state"
+                    name="state"
+                    type="text"
+                    readonly="readonly"
+                    class="w-full px-2 py-1 bg-yellow-50"
+                >
+            </div>
+            <div class="w-1/12 ml-2">
+                <label class="ml-2 text-xs text-gray-500 font-semibold tracking-widest">
+                    Zip
+                </label>
+                <input
+                    id="zip"
+                    name="zip"
+                    type="text"
+                    readonly="readonly"
+                    class="w-full px-2 py-1 bg-yellow-50"
+                >
+            </div>
+        </div>
+        <button 
+            type="submit"
+            class="mt-2 px-2 py-1 bg-white text-green-500 border-2 border-green-400 rounded-md"
+        >
+            Submit
+        </button>
+    </form>
+</div>
+
+<script>
```

Comment:
@drewroberts really need to pin down what the base layout supports.  Injecting javascript inline w/HTML is bad practice.  There's also AMP considerations in all of this.

Reply:
That sounds like a better approach. I've done the stack approach for scripts in the past and like how clean it is.

Reply:
The pros of the Javascript version is that 'session token handling' is taken care of by the used Google classes.  Comparing to the Livewire version, I am not sure how well the helper Str::uuid() will handle session tokens.

Reply:
If the livewire version doesn't work, that would be the one to kill.  Why offer both if you're only going to use one?  If its only temporary while trying to fix the livewire version, that's ok.

Reply:
For billing, a Google "session" starts with 'autocomplete' requests, and ends with 'place details' request.  If a new _unique_ session token isn't passed for the next Google session, we get billed for every single action following.  I'm not sure if the _Str::uuid()_ can accomplish that across all 'Project instances'.

Reply:
I'm assuming Str:uuid() is based on [ramsey/uuid](https://uuid.ramsey.dev/en/latest/introduction.html) 

> The probability of duplicating a UUID is close to zero, so they are a great choice for generating unique identifiers in distributed systems.  

https://stackoverflow.com/questions/52751438/laravel-uuids-and-their-uniqueness